src/theory/theory.h

```diff
@@ -440,18 +440,12 @@ class Theory : protected EnvObj
     EFFORT_LAST_CALL = 200
   }; /* enum Effort */
 
-  static inline bool standardEffortOrMore(Effort e) CVC5_CONST_FUNCTION
-  {
-    return e >= EFFORT_STANDARD;
-  }
-  static inline bool standardEffortOnly(Effort e) CVC5_CONST_FUNCTION
+  static bool standardEffortOrMore(Effort e) { return e >= EFFORT_STANDARD; }
+  static bool standardEffortOnly(Effort e)
```

Comment:
Maybe this should be `constexpr` now?

Reply:
Quick note: We have discussed this offline and came to the conclusion that `constexpr` does not apply here (and that the two methods `standardEffortOrMore()` and `standardEffortOnly()` are not really used).